Makefile

```diff
@@ -69,14 +69,16 @@ make build          # compile and link
   STATIC=1            # build as static lib
   LITE=1              # build RediSearchLight
   DEBUG=1             # build for debugging
-  STATIC_LIBSTDCXX=0  # link libstdc++ dynamically (default: 1)
   NO_TESTS=1          # disable unit tests
   WHY=1               # explain CMake decisions (in /tmp/cmake-why)
   FORCE=1             # Force CMake rerun (default)
   CMAKE_ARGS=...      # extra arguments to CMake
   VG=1                # build for Valgrind
   SAN=type            # build with LLVM sanitizer (type=address|memory|leak|thread) 
   SLOW=1              # do not parallelize build (for diagnostics)
+  GCC=1               # build with GCC (default unless Sanitizer)
+  CLANG=1             # build with CLang
```

Comment:
Do we want to expose `CLANG=1` for non-mac and and support a "build matrix"?

Reply:
Yes, this is relevant (implicity) to sanitizer, and if you make clang available on your system (e.g. via `readies/bin/getclang`).
I'm considering doing an implicit variant for non-default compiler selections.
PS there's also INTELCC under way.

Reply:
:astonished: